src/lib/components/composites/statistics/ProjectInformation.svelte

```diff
@@ -1,9 +1,22 @@
 <script lang="ts">
-    import type { ProjectInformationInterface } from "$lib/model/component-interfaces";
     import { Skeleton } from "$lib/components/primitives/skeleton";
     import { pluralize } from "$lib/utils/common-helper";
     import ErrorIndicator from "$lib/components/composites/utils/ErrorIndicator.svelte";
 
+    /**
+     * Contains interfaces or types that specify the interface of components.
+     * These are not defined locally so that these types can be used at the point where the data for the components is loaded or created too.
+     */
+    export interface ProjectInformationInterface {
```

Comment:
Don't just copy it 😅 
```suggestion
    export interface ProjectInformationInterface {
```

Reply:
You're right. This doesn't belong here!